### DISCUSSION TITLE
Remove 'href.li' redirect

### DISCUSSION THREAD

**Discussion by StarChaser** [AprilSylph/XKit-Rewritten#804] [Help Desk]
Any chance of an addon to murder the href.li redirect? Every so often they do something to the site that adds that and breaks links, everything goes to the site 'href.li' which doesn't actually exist, instead of being redirected to whatever the link actually is. 
example link: 
https://href.li/?https://babylonbee.com/news/stephen-king-sues-drag-queens-for-infringing-copyright-of-hideous-clowns-preying-on-children/
Note that if you remove https://href.li/? the link works normally. I assume this is some sort of attempt at tracking clicks or something.

**Comment by alleycatboy** [comment]
I've got a something working for this, it's definitely possible!

**Reply by StarChaser** [reply]
Excellent, thanks!

**Comment by AprilSylph** [comment]
> I assume this is some sort of attempt at tracking clicks or something.

nope, href.li is a referrer-stripping service, so that websites can't tell that you clicked a link on Tumblr to get there.

**Reply by StarChaser** [reply]
Ah, so. I complained about it because at the time, and every so often, the redirect doesn't work, just get 'href.li does not exist'.
